tests/bettertransformer/test_bettertransformer_audio.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+
+import numpy as np
+import torch
+from transformers import AutoFeatureExtractor, AutoModel, AutoProcessor
+
+from optimum.bettertransformer import BetterTransformer
+from testing_bettertransformer_utils import BetterTransformersTestMixin
+
+
+ALL_AUDIO_MODELS_TO_TEST = [
+    "openai/whisper-tiny",
+    "patrickvonplaten/wav2vec2_tiny_random",
+    "ybelkada/hubert-tiny-random",
+]
+
+
+class BetterTransformersWhisperTest(BetterTransformersTestMixin, unittest.TestCase):
+    r""" """
```

Comment:
Docstring intended?

Reply:
Fixed in [a958177](https://github.com/huggingface/optimum/pull/470/commits/a958177cb8f193ac54e8174fab997a31b85d3750)